src/en/web/design/layout.md

```diff
@@ -102,11 +102,35 @@ It is very difficult to associate the themes ("par region", "par genre"...) and
 
 
 
-## Define sensitive areas of sufficient size
+## Minimum target size
 
-**Target:** everyone in particular, people with motor or visual disability and mobility.  
+**Target:** Everyone, especially people with visual impairments and those with motor or mobility disabilities.  
 **When:** during design and development.
 
 **Description:**  
-Each sensitive area must have a sufficient size (minimum 9mm width and height).
-In addition, the sensitive areas must be sufficiently spaced from each other (about 2mm minimum).
+- Interactive elements must have a sufficiently large target size or be adequately spaced to be easily activated, reducing the risk of errors.
+- Interactive elements must measure at least 24 pixels height by 24 pixels width. It is essential to consider the entire click or touch area, not just the text or image.
+
+**Checklist:**
+- Verify the dimensions of interactive elements.
+- Ensure that interactive elements meet a minimum target size of 24 pixels height by 24 pixels width, under all display conditions regardless of zoom level or viewport size.
+
+**Compliant exceptions:**
+- If interactive elements have a target size smaller than the minimum of 24x24 pixels but are positioned so that a circle with a diameter of 24 pixels centered on each does not overlap with other interactive elements.
```

Comment:
```suggestion
- If interactive elements have a target size smaller than the minimum of 24x24 pixels but are positioned so that a circle with a diameter of 24 pixels centered on the target does not overlap with another target or the circle for an undersized target.
```